protocol/v2/ssv/testing/runner.go

```diff
@@ -80,93 +81,95 @@ var baseRunner = func(
 	config.OperatorSigner = opSigner
 	config.SignatureVerifier = spectestingutils.NewTestingVerifier()
 
-	contr := specqbft.NewController(
+	contr := testing.NewTestingQBFTController(
```

Comment:
I'm pretty sure this wasn't used before and the code might not be ready there. can you give it a double check?